tox.ini

```diff
@@ -32,13 +32,11 @@ changedir = .tmp/{envname}
 description =
     run tests
     devdeps: with the latest developer version of key dependencies
-    olddeps: with medium old versions of key dependencies
```

Comment:
If there are concerns that the range of supported `numpy` versions is so wide that the `devdeps` and `oldestdeps` CI jobs are not enough then I suggest following the advice of [SPEC 0](https://scientific-python.org/specs/spec-0000/) and limiting the range of supported versions.